Antidote/QRScannerController.m

```diff
@@ -179,10 +191,14 @@ - (void)createCaptureSession
 
 - (void)createBarButtonItems
 {
-    self.navigationItem.leftBarButtonItem =
-        [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemCancel
-                                                      target:self
-                                                      action:@selector(cancelButtonPressed)];
+    UIButton *cancelButton = [UIButton buttonWithType:UIButtonTypeCustom];
+    [cancelButton.titleLabel setTextColor:[UIColor yellowColor]];
```

Comment:
You can't change button text color (@"Cancel") for custom color if you use UIBarBattinSystemItemCancel. So i used custom UIButton for setup text color like application color scheme.